irony.el

```diff
@@ -426,6 +428,7 @@ If no such file exists on the filesystem the special file '-' is
       (display-warning 'irony "Performance will be bad because a\
  pipe delay is set for this platform (see variable\
  `w32-pipe-read-delay')."))))
+  (irony-xref--enter)
```

Comment:
Should this be optional?
I think some people use irony for completion only, and sometime use rtags or something else for cross reference.